### ISSUE TITLE
Documentation: polls integration missing templates

### ISSUE THREAD

**Issue by geoidesic** [divio/django-cms#6394]
### Summary

http://docs.django-cms.org/en/latest/introduction/apphooks.html#create-an-apphook

This doesn't work as is. It's not clear where these views are defined in the tutorial (I hunted and couldn't find them anywhere).

### Expected behaviour
Should provide the missing templates or should refer to views that are actually part of the polls 3rd party app such as:

`IndexView`
`DetailView`
`ResultsView`

### Actual behaviour
Tutorial seems incomplete. On `runserver` I get the following error:
 ```
from polls.views import PollListView, PollDetailView
ImportError: cannot import name 'PollListView'
```

### Environment

* Python version: 3.6.4
* Django version: 1.11.11
* django CMS version: 3.5.1

**Comment by evildmp** [comment]
Documentation/polls application mismatch.

**Comment by ZippoLag** [comment]
Hi, I'm stuck at the same step (following this tutorial http://docs.django-cms.org/en/latest/how_to/namespaced_apphooks.html#multi-apphook), what's the quickest way to fix this? Should I change the version of the polls app I downloaded? (by tutorial guidance, I used the git repo directly).

For now, I'm using "DetailView" instead of "PollDetailView" and "IndexView" instead of "PollListView" because that's what I'm guessing's write according to the code currently on the repo (https://github.com/divio/django-polls/blob/master/polls/views.py), but the docs should be updated by some who's sure what should be in there I guess.
